compliance/collection/inventory/convert.go

```diff
@@ -0,0 +1,110 @@
+package inventory
+
+import (
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/uuid"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+)
+
+// ToNodeInventory converts a NodeInventory response to a native storage.NodeInventory version
+func ToNodeInventory(resp *scannerV1.GetNodeInventoryResponse) *storage.NodeInventory {
+	ni := &storage.NodeInventory{
+		NodeId:     uuid.Nil.String(),
+		NodeName:   resp.GetNodeName(),
+		ScanTime:   timestamp.TimestampNow(),
+		Components: toStorageComponents(resp.GetComponents()),
+		Notes:      convertNotes(resp.GetNotes()),
+	}
+	return ni
+}
+
+func toStorageComponents(c *scannerV1.Components) *storage.NodeInventory_Components {
+	if c == nil {
+		return nil
+	}
+
+	var namespace string
+	if c.GetNamespace() == "" {
+		namespace = "unknown"
```

Comment:
Why setting namespace to a hardcoded value such as `unknown`? I'm not sure how these components are bound to namespaced resources, but global resources usually have empty (`""`) namespaces. And that's a valid value for this field in k8s.  

I think leaving the namespace with the exact same value when converting (even if empty) would make more sense. 

If there is a non obvious reason, could you add a comment about it then?